app/models/public_body_change_request.rb

```diff
@@ -33,9 +33,13 @@ class PublicBodyChangeRequest < ActiveRecord::Base
   validate :user_email_format, :unless => proc{ |change_request| change_request.user_email.blank? }
   validate :body_email_format, :unless => proc{ |change_request| change_request.public_body_email.blank? }
 
-  scope :new_body_requests, :conditions => ['public_body_id IS NULL'], :order => 'created_at'
-  scope :body_update_requests, :conditions => ['public_body_id IS NOT NULL'], :order => 'created_at'
-  scope :open, :conditions => ['is_open = ?', true]
+  scope :new_body_requests, -> {
+    where(public_body_id: nil).order("created_at")
+  }
+  scope :body_update_requests, -> {
+    where("public_body_id <> ?", nil).order("created_at")
```

Comment:
This doesn't seem to have the same behaviour:

```ruby
[3](main)> PublicBodyChangeRequest.body_update_requests
  PublicBodyChangeRequest Load (0.3ms)  SELECT "public_body_change_requests".* FROM "public_body_change_requests" WHERE (public_body_id <> NULL) ORDER BY created_at
=> []
[4](main)> PublicBodyChangeRequest.where("public_body_id IS NOT NULL").order("created_at")
  PublicBodyChangeRequest Load (0.4ms)  SELECT "public_body_change_requests".* FROM "public_body_change_requests" WHERE (public_body_id IS NOT NULL) ORDER BY created_at
=> [#<PublicBodyChangeRequest id: 1, user_email: nil, user_name: nil, user_id: 7, public_body_name: nil, public_body_id: 7, public_body_email: "wehbcehuc@example.com", source_url: "", notes: "", is_open: true, created_at: "2015-09-23 09:48:42", updated_at: "2015-09-23 09:48:42">, #<PublicBodyChangeRequest id: 2, user_email: nil, user_name: nil, user_id: 10, public_body_name: nil, public_body_id: 3, public_body_email: "blahidbbd@example.com", source_url: "", notes: "", is_open: true, created_at: "2015-09-29 13:36:15", updated_at: "2015-09-29 13:36:15">]
```

Would be worth adding a spec for this.